### PR TITLE
Conflict branch

### PR DESCRIPTION
**Pull request by merkurt2001**

### CODE REVIEW

**Review by merkurt2001 — COMMENTED** [review]
ok

**Review by yalcin-yildiz — APPROVED** [review]

File: src/test/java/YakupC.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 	//commit 1
 
-	//commit 2
```
Comment (merkurt2001) [inline src/test/java/YakupC.java:5]: please add this line again

**Review by Yck61 — CHANGES_REQUESTED** [review]
ok

**Review by merkurt2001 — COMMENTED** [review]
ddd

**Review by Yck61 — CHANGES_REQUESTED** [review]
ok

**Review by Yck61 — CHANGES_REQUESTED** [review]
test.java

**Review by Yck61 — CHANGES_REQUESTED** [review]
YakupC.java

**Review by Yck61 — APPROVED** [review]
a

**Comment by yalcin-yildiz** [comment]
please check again and send after that

**Pull request closed without merging**